specification/sql/resource-manager/Microsoft.Sql/preview/2024-02-01-preview/DatabaseBilling.json

```diff
@@ -0,0 +1,124 @@
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 InvalidBillingVersion - Currently only moving from version 1 to 2 is supported.\n\n * 404 ServerNotInSubscriptionResourceGroup - Specified server does not exist in the specified resource group and subscription.\n\n * 404 OperationIdNotFound - The operation with Id does not exist.\n\n * 404 ResourceGroupNotFound - The resource group provided was not found.* 404 DatabaseDoesNotExist - The requested database was not found\n\n 409 OperationCancelled - The operation has been cancelled by user.\n\n * 409 OperationInterrupted - The operation on the resource could not be completed because it was interrupted by another operation on the same resource.\n\n * 429 SubscriptionTooManyRequests - Requests beyond max requests that can be processed by available resources.\n\n * 500 OperationTimedOut - The operation timed out and automatically rolled back. Please retry the operation.\n\n * 503 ServiceTemporarilyUnavailable - Feature temporarily unavailable.\n\n * 503 TooManyRequests - Requests beyond max requests that can be processed by available resources.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Request a specific billing version for a database.": {
+            "$ref": "./examples/UpdateBillingVersion.json"
+          }
+        }
+      }
+    }
+  },
+  "parameters": {
+    "DatabaseNameParameter": {
+      "name": "databaseName",
+      "in": "path",
+      "description": "The name of the database.",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method",
+      "minLength": 1,
+      "pattern": "^[-\\w\\._]+$"
+    },
+    "ServerNameParameter": {
+      "name": "serverName",
+      "in": "path",
+      "description": "The name of the server.",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method",
+      "minLength": 1,
+      "pattern": "^[-\\w\\._]+$"
+    }
+  },
+  "definitions": {
+    "DatabaseBillingVersion": {
+      "type": "integer",
+      "format": "int32",
+      "description": "The pricing model to use when calculating billing for this database.",
+      "enum": [
+        1,
+        2
+      ],
+      "x-ms-enum": {
+        "name": "DatabaseBillingVersion",
+        "modelAsString": true
+      }
+    },
+    "DatabaseBillingVersionResponse": {
+      "type": "object",
+      "description": "A response containing a billing version.",
+      "properties": {
+        "properties": {
+          "billingVersion": {
+            "$ref": "#/definitions/DatabaseBillingVersion"
+          },
+          "name": {
+            "type": "string"
+          },
+          "id": {
+            "type": "string"
+          }
```

Comment:
single resource GET responses are required to have these, but they should normally be by inheritance from e.g. ProxyResource